.editorconfig

```diff
@@ -4,9 +4,9 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 charset = utf8
-trim_trailing_whitespace = true
+trim_trailing_whitespace = false
 
```

Comment:
@ghar [see my first bullet point](https://github.com/neomutt/neomutt/pull/439#issue-210464653). That's just a temporary fix, until the clang-format has run on our codebase - which is, according to @flatcap, to ready at the moment.